bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -74,6 +85,9 @@ public function __construct(
         $this->tokenStorage = $tokenStorage;
         $this->fileUploader = $fileUploader;
         $this->userManager = $userManager;
+        $this->swiftMailer = $swiftMailer;
+        $this->twig = $twig;
+        $this->translator = $translator;
```

Comment:
Missing in top of class.

Reply:
Please explain, "top of class", looks good to me.

Reply:
$this->translator is not declared at the top of the class.

`private $translator;`

Reply:
Ahh, true